spec.html

```diff
@@ -22,10 +22,27 @@ <h1>Introduction</h1>
   <h1>Syntax</h1>
 
   <emu-grammar>
-      LogicalORExpression[In, Yield, Await] :
-        LogicalANDExpression[?In, ?Yield, ?Await]
-        LogicalORExpression[?In, ?Yield, ?Await] `||` LogicalANDExpression[?In, ?Yield, ?Await]
-        <ins>LogicalORExpression[?In, ?Yield, ?Await] `??` LogicalANDExpression[?In, ?Yield, ?Await]</ins>
+      ConditionalExpression[In, Yield, Await] :
+        <del>LogicalORExpression[?In, ?Yield, ?Await]</del>
+        <del>LogicalORExpression[?In, ?Yield, ?Await] `?` AssignmentExpression[+In, ?Yield, ?Await] `:` AssignmentExpression[?In, ?Yield, ?Await]</del>
```

Comment:
I think we can remove these two lines.

Reply:
They're `<del>`d (I made the same mistake).

Reply:
Ahhhhhh